### ISSUE TITLE
wrong import is generated for hyphenated package

### ISSUE THREAD

**Issue by nkovacs** [maxbrunsfeld/counterfeiter#73]
Before the source code is passed to goimports, counterfeiter generates this import into fixturesfakes/fake_imports_go_hyphen_package.go (from https://github.com/maxbrunsfeld/counterfeiter/blob/b8c427d36da1fb0067515646ab4063195f8f89e0/fixtures/imports_go_hyphen_package.go):

```
go_hyphenpackage "github.com/maxbrunsfeld/counterfeiter/fixtures/go-hyphenpackage"
```

However, that's not how the package is used in the code. It's used as `hyphenpackage`, so goimports removes this unused import, and inserts a new one. Since I have a fork, and the fork's name is shorter, it selects the fixture from my fork.

**Comment by nkovacs** [comment]
The bug is caused by this line: https://github.com/maxbrunsfeld/counterfeiter/blob/366cd53ac5d33e99b97386e04584238fbdc98729/locator/locator.go#L223

This is not correct. While usually the package name and the directory name are the same, they don't have to be, and you don't have to use an alias for the import if they're not (although goimports will add one to make the code less confusing).

**Comment by joefitzgerald** [comment]
I think this will be fixed by #94, please check out that PR.

**Comment by joefitzgerald** [comment]
We now alias all imports, so that we never end up with mismatches.
